data-claims-event-service/src/test/java/uk/gov/justice/laa/dstew/payments/claimsevent/validation/JsonSchemaValidatorTest.java

```diff
@@ -788,18 +1220,29 @@ void validateClaimForInvalidDataTypes(
       "transferDate, '2025-00-10', 'transfer_date: does not match the regex pattern ^(19\\d{2}|20\\d{2}|2100)-(0?[1-9]|1[0-2])-(0?[1-9]|[12][0-9]|3[01])$ (provided value: 2025-00-10)'",
       "transferDate, '2025-02-32', 'transfer_date: does not match the regex pattern ^(19\\d{2}|20\\d{2}|2100)-(0?[1-9]|1[0-2])-(0?[1-9]|[12][0-9]|3[01])$ (provided value: 2025-02-32)'"
     })
-    void validateClaimIndividualInvalidField(
+    void validateClaimIndividualInvalidFieldLegalHelpMediation(
```

Comment:
can the @CsvSource be moved to @CsvFileSource so that tests will be easier to read ?